packages/gatsby-theme-docs/src/components/cards.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { designSystem, ContentNotifications } from '@commercetools-docs/ui-kit';
+
+// Explanation about the following sizes:
+// https://github.com/commercetools/commercetools-docs-kit/pull/427#discussion_r425442556
+const cardNarrowMinWidth = '242px';
+const cardRegularMinWidth = '328px';
+const CardsContainer = styled.div`
+  display: grid;
+  gap: ${designSystem.dimensions.spacings.m};
+  grid-auto-columns: 1fr;
+  grid-template-columns: ${(props) =>
+    `repeat(auto-fill, minmax(${
+      props.narrow ? cardNarrowMinWidth : cardRegularMinWidth
+    }, 1fr))`};
+`;
+
+const Cards = (props) => {
+  try {
+    return (
+      <CardsContainer {...props}>
+        {React.Children.map(props.children, (child) => {
+          if (!child.props || child.props.mdxType !== 'Card') {
```

Comment:
Note that by using `mdxType` it implies that the `Cards` component can only be used within MDX files.
It would be better to check for example if the child component `displayName` is `Card`.